internal/device_managers/deviceManagers.go

```diff
@@ -0,0 +1,28 @@
+package device_managers
```

Comment:
How about `partitioners`?

Reply:
Yeah thats a better name; will do!

Reply:
I know renames can be confusing in prs, so i did some commit juggling and it has been renamed; I hope I did it in a way that made sense.